Ibnul Ekram

```diff
@@ -0,0 +1,20 @@
+import junit.framework.Assert;
+
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+public class NFLTest
+
+{
+@Test
+public void TestTitle()
+
+{
+WebDriver dr= new FirefoxDriver();
+dr.get("http://scores.espn.go.com/nfl/scoreboard");
```

Comment:
Hi there,
I have updated it. Here is the updated script

import junit.framework.Assert;

import org.junit.Test;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class NFLSCORE

{
@Test
public void TestTitle()

{
WebDriver dr= new FirefoxDriver();
dr.get("http://scores.espn.go.com/nfl/scoreboard");
        //Clicking on NFL link
dr.findElement(By.name("&lpos=sitenavdefault&lid=sitenav_nfl")).click();
        //Clicking on Score link
dr.findElement(By.name("&lpos=subnav&lid=subnav_NFL_Scoreboard")).click();

String ptitle=dr.getTitle();
Assert.assertTrue("Title not found!", ptitle.contains("NFL Scoreboard"));

}
}


Regards,

Ibnul Ekram
25 49 82 ST., East Elmhurst, NY 11370
Email: ibnulekram@gmail.com
Phone: 9292356101

On Tue, Jan 13, 2015 at 1:26 PM, Brian R. Jackson <notifications@github.com>
wrote:

> In Ibnul Ekram
> <https://github.com/jaxzin/fizzbuzz-selenium/pull/21#discussion-diff-22882489>
> :
>
> > @@ -0,0 +1,20 @@
> > +import junit.framework.Assert;
> > +
> > +import org.junit.Test;
> > +import org.openqa.selenium.WebDriver;
> > +import org.openqa.selenium.firefox.FirefoxDriver;
> > +public class NFLTest
> > +
> > +{
> > +@Test
> > +public void TestTitle()
> > +
> > +{
> > +WebDriver dr= new FirefoxDriver();
> > +dr.get("http://scores.espn.go.com/nfl/scoreboard");
>
> Can you update this test to start on espn.go.com and click on the link
> for the NFL scoreboard?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jaxzin/fizzbuzz-selenium/pull/21/files#r22882489>.
>